composeApp/src/commonMain/kotlin/designsystem/login/FacebookSignUpHyperlink.kt

```diff
@@ -0,0 +1,35 @@
+package designsystem.login
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import designsystem.HyperLinkText
+import dev.icerock.moko.resources.compose.painterResource
+import dev.icerock.moko.resources.compose.stringResource
+import instaclone.resources.MR
+
+@Composable
+fun FacebookSignUpHyperlink() {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center, modifier = Modifier.padding(20.dp)
+    ) {
+        Image(
+            painter = painterResource(MR.images.facebook_logo),
+            contentDescription = "Facebook logo"
+        )
+        HyperLinkText(
+            modifier = Modifier.padding(start = 10.dp),
+            text = stringResource(MR.strings.facebook_login), fontWeight = FontWeight.W600
+        ) {
+            /* TODO ADD NAVIGATION ACTION */
```

Comment:
**#question**

Can you create an issue to address that behavior in the future?

Reply:
Yep!